### PR TITLE
Disable static routes when service type is set as NodePortLocal

### PR DESCRIPTION
**Pull request by sudswasavi**
This commit disables programming static routes if the service type
is set to NodePortLocal

In addition to this, a validation check is added that verifies if the cniPlugin is 'antrea' if the serviceType is set as NodePortLocal. Else AKO is rebooted.

### CODE REVIEW

**Review by monotosh-avi — APPROVED** [review]

File: helm/ako/values.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ AKOSettings:
   deleteConfig: "false" # Has to be set to true in configmap if user wants to delete AKO created objects from AVI 
   disableStaticRouteSync: "false" # If the POD networks are reachable from the Avi SE, set this knob to true.
   clusterName: "my-cluster" # A unique identifier for the kubernetes cluster, that helps distinguish the objects for this cluster in the avi controller. // MUST-EDIT
-  cniPlugin: "" # Set the string if your CNI is calico or openshift. enum: calico|canal|flannel|openshift 
+  cniPlugin: "" # Set the string if your CNI is calico or openshift. enum: calico|canal|flannel|openshift|antrea
```
Comment (akshayhavile) [inline helm/ako/values.yaml:19]: @sudswasavi : I think we need to do same changes at ako-operator side yaml file as well.

Comment (akshayhavile) [reply helm/ako/values.yaml:19]: Also comment for L7Settings:serviceType  should include "NodePortLocal"

Comment (sudswasavi) [reply helm/ako/values.yaml:19]: Thanks. Added the NodePortLocal change. For AKO-operator would, address in a separate PR.

**Comment by sudswasavi** [comment]
build ako

**Comment by monotosh-avi** [comment]
build ako

**Comment by sudswasavi** [comment]
build ako

**Comment by sudswasavi** [comment]
build ako

**Comment by hemantavi** [comment]
build ako

**Review by monotosh-avi — APPROVED** [review]

**Review by chauhanshubham — APPROVED** [review]

**Pull request merged**